thirtyinch/build.gradle

```diff
@@ -49,3 +55,16 @@ configurePublish {
     artifactId = 'thirtyinch'
     desc = 'a Model View Presenter library for Android'
 }
+
+task copyLintJar(type: Copy) {
+    from(configurations.lintChecks) {
+        rename { String fileName -> 'lint.jar' }
+    }
+    into 'build/intermediates/lint/'
```

Comment:
Is it common to move the lint check into the final AAR? 🤔 
What about to publish a separate jar/artifact and make this optional? 🤔

Reply:
Having custom lint checks for a library is only really possible since AGP v3.0. Before you had to manually copy a lint.jar to specific folder and so on.
So not a lot libs ship their own lint checks with the AAR atm.

But in my view it is way more convenient if you have them out-of-the-box. If you do not want them, you could deactivate them in the Android Studio preferences anyway.